src/include/stir/ProjData.h

```diff
@@ -120,10 +120,14 @@ class ProjData : public ExamData
   //! Destructor
   virtual ~ProjData() {}
   //! Get shared pointer to proj data info
+  inline shared_ptr<const ProjDataInfo>
+    get_proj_data_info_sptr() const;
+  //! Get shared pointer to proj data info
   /*! \warning Use with care. If you modify the object in a shared ptr, everything using the same
     shared pointer will be affected. */
   inline shared_ptr<ProjDataInfo>
-    get_proj_data_info_sptr() const;
+    get_proj_data_info_sptr();
```

Comment:
hmmm. But this exposes everything anyway. Moreover, we have no clue which one will be called. I thought the plan was to remove this